src/kendo.view.js

```diff
@@ -75,6 +75,7 @@ var __meta__ = { // jshint ignore:line
             that.model = options.model;
             that._wrap = options.wrap !== false;
             this._evalTemplate = options.evalTemplate || false;
+            this._useWithBlock = options.useWithBlock ?? true;
```

Comment:
The `??` operator is not cross-browser. Plus, as the useWithBlock is handled by the template, you can let the template default to true. 

`this._useWithBlock = options.useWithBlock;`